src/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -216,6 +216,72 @@ elementFormDefault="qualified">
               </xs:documentation>
             </xs:annotation>
           </xs:attribute>
+          <xs:attribute name="IncludeAssets" type="xs:string" use="optional">
+            <xs:annotation>
+              <xs:documentation>
+                <!-- _locID_text="PackageReference_Attribute_IncludeAssets" _locComment="" -->Assets to include from this reference
```

Comment:
Do these match the NuGet docs for these? Should we link to those docs in the loc comment to ease translation consistency?